themes/default/content/docs/reference/cli/_index.md

```diff
@@ -107,3 +107,19 @@ Then reload your shell:
 ```shell
 exec $SHELL -l
 ```
+  
+### Fish
+
+The `pulumi gen-completion fish` command self-generates its own CLI script. You can save the output to a file or use it in the current session:
+
+```shell
+pulumi gen-completion fish | source
+```
+
+To load the completions for each session, you need to save the completion to a file
+
+```bash
+pulumi gen-completion fish > ~/.config/fish/completions/yourprogram.fish`
```

Comment:
My bash-fu isn't super strong, but is there supposed to be a backtick at the end of this statement?

Reply:
Oh, good spot! I think that's probably a typo.

Reply:
typo from before - good catch!